packages/services/src/services/Analytics/Analytics.js

```diff
@@ -214,6 +214,86 @@ class AnalyticsAPI {
       }
     }
   }
+
+  /**
+   * Sends video player metrics data
+   *
+   * @param {string} data event data from the video player
+   *
+   * @example of the `data` object structure:
+   * data = {
+   *    playerType: "kaltura"
+   *    title: "Folgers Coffee"
+   *    currentTime: 1
+   *    duration: 60
+   *    playerState: 1
+   *    videoId: "0_uka1msg4"
+   * }
```

Comment:
The JSDoc output comes out like this:

![Screen Shot 2020-03-18 at 10 24 56 AM](https://user-images.githubusercontent.com/1641214/76970815-cb4e9f80-6902-11ea-84a6-ce9398a97720.png)

Should probably do a full example of using the method. You can also possibly add the data object itself under the `@param` too to supplement:

```
@param {{ playerType: string, title: string, currentTime: int, duration: int, playerState: int, videoId: string}} data Event data from the video player
```

Reply:
Added a full example in the example of JSDOCs, decided not to add the object under the `@param` - seems to be overkill with the full example already in there